src/main/kotlin/no/nav/familie/tilbake/forvaltning/ForvaltningService.kt

```diff
@@ -239,14 +247,12 @@ class ForvaltningService(
             )
```

Comment:
"Finnes ikke data i systemet... => Finnes ikke "uarkiverte" kravgrunnlag for (...)